docs/notebooks/search_formele_stratigrafie.ipynb

```diff
@@ -22,8 +22,10 @@
   },
```

Comment:
We should add this to the other Notebooks in the upcoming code sprint as well

Reply:
Agreed, we can do that in separate PR's I think, allowing this one to be merged already.